docs/src/main/paradox/docs/delta/api/current/views/aggregated-es-view-api.md

```diff
@@ -3,11 +3,17 @@
 This view is an aggregate of ElasticSearchViews. The view itself does not create any index, but it references the 
 already existing indices of the linked ElasticSearchViews.
 
+@@@ note
+
+From Delta version 1.5.0,  `AggregateElasticSearchView` can point to other `AggregateElasticSearchView`s. 
+
+@@@
+
 When performing queries on the `_search` endpoint, this view will make use of the 
 @link:[multi-index](https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-index.html){ open=new } query 
 capabilities of ElasticSearch in order to select the indices of every view present on this aggregate view.
 
-If the caller does not have the permission `views/query` on all the projects defined on the aggregated view, only a 
+If the caller does not have the required permission on all the views defined on the aggregated view, only a 
```

Comment:
I would suggest to have something similar to the aggregate sparqlview documentation:
> If the caller does not have the permission `views/query` (or from v1.5, the user-defined permission) on all the views defined on the aggregated view, only a subset of indices (or none) will be selected, respecting the defined permissions.